tag_manager/custom_popup_examples.rst

```diff
@@ -0,0 +1,50 @@
+.. highlight:: js
+.. default-domain:: js
+
+.. versionadded:: 10.1
+
+Custom popup template implementation examples
+=============================================
+
+Introduction
+------------
+Since version 10.1 of PPAS there is a possiblity of creating a `Custom popup` tag template. We decided to share some example implementations that can be further modified and expanded.
+
+Example 1
+----------
+Preview:
+    .. image:: /_static/images/tm_popup_examples/example-popup-1.png
+        :alt: Custom popup example 1
+
+.. note::
+    Handling of the close button is provided in ContainerJS, as long as the class name ``ppms-popup-close-button`` is unchanged. JavaScript code to handle `Subscribe now` button needs to be provided.
```

Comment:
not sure if clients are aware of ContainerJS keyword

Reply:
Change to a more general `provided out of the box`